src/index.html

```diff
@@ -10,13 +10,70 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav-bar">
+          <li class="nav-list-elem">
+            <a
+              href="#"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav-list-elem margin-left-20">
+            <a href="#">Samsung</a>
+          </li>
+          <li class="nav-list-elem margin-left-20">
+            <a href="#">Smartphones</a>
+          </li>
+          <li class="nav-list-elem margin-left-20">
+            <a
+              href="#"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav-list-elem margin-left-20">
+            <a href="#">Gadgets</a>
+          </li>
+          <li class="nav-list-elem margin-left-20">
+            <a href="#">Tablets</a>
+          </li>
+          <li class="nav-list-elem margin-left-20">
+            <a href="#">Photo</a>
+          </li>
+          <li class="nav-list-elem margin-left-20">
+            <a href="#">Video</a>
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. Instead of 'nav-list-elem margin-left-20', consider using a class that describes the role or content, such as 'nav__item'.